---
layout: default
title: aries-agent-test-harness
parent: Hyperledger
grand_parent: Pull Requests
has_children: false
permalink: /pull-requests/hyperledger/aries-agent-test-harness
---

# aries-agent-test-harness <span class="fs-3 right-align">[GitHub](https://github.com/hyperledger/aries-agent-test-harness){: .btn .mr-4 }</span>


<div>
    <table>
        <tr>
            <td>
                PR <a href="https://github.com/hyperledger/aries-agent-test-harness/pull/524" class=".btn">#524</a>
            </td>
            <td>
                <b>
                    Changes by create-pull-request action
                </b>
            </td>
        </tr>
        <tr>
            <td>
                
            </td>
            <td>
                Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action
            </td>
        </tr>
    </table>
    <div class="right-align">
        Created At 2022-07-03 04:30:53 +0000 UTC
    </div>
</div>

<div>
    <table>
        <tr>
            <td>
                PR <a href="https://github.com/hyperledger/aries-agent-test-harness/pull/522" class=".btn">#522</a>
            </td>
            <td>
                <b>
                    Try fix for error happening on GHA runs of generate results, and debugging for this case
                </b>
            </td>
        </tr>
        <tr>
            <td>
                
            </td>
            <td>
                Not clear why the error is occurring -- perhaps something to do with Allure results. Added a potential fix and debug info.
            </td>
        </tr>
    </table>
    <div class="right-align">
        Created At 2022-06-28 22:29:54 +0000 UTC
    </div>
</div>

